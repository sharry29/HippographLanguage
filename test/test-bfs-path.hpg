int main() {
	graph<int> g = [1 -()- 2 -()- 3; 1 -()- 4];
	graph<int> g_sub1 = g.bfs(1);
	g_sub1.print();
}