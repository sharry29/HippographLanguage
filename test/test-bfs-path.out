1:null -> []
2:null -> []
4:null -> []
3:null -> []